src/components/FilterPanel.jsx

```diff
@@ -43,8 +49,23 @@ export default function FilterPanel() {
       />
     ),
   };
+
   return (
     <Box>
+      <Stack
+        direction="row"
+        justifyContent="space-between"
+        alignItems="center"
+        sx={{ display: { md: 'none' }, my: 1 }}>
```

Comment:
Instead of a `<Stack>` I suggest using a [`<Toolbar>`](https://mui.com/api/toolbar/) instead. It should come with the margins you need and display things in a row.